app/src/main/java/org/merakilearn/ui/onboarding/LoginViewModel.kt

```diff
@@ -18,4 +18,12 @@ class LoginViewModel(private val applicationRepo: ApplicationRepo) : ViewModel()
     fun logOut() = liveData {
         emit(applicationRepo.logOut())
     }
+
+    fun deleteFile(first: String) = liveData {
+        emit(applicationRepo.deleteFile(first))
+    }
+
+    val fetchSavedFile = liveData {
```

Comment:
same goes here, this should not return anything

Reply:
This is not function returning live data. It's a variable which is assigned with live data